jobs/forms.py

```diff
@@ -5,10 +5,17 @@
 
 class JobForm(forms.ModelForm):
 
-    company_name = forms.CharField(max_length=150, required=True)
+    company_name = forms.CharField(label='Company',
+                                   max_length=500,
+                                   required=True)
+    website = forms.URLField(label='Company website', required=True)
 
     class Meta:
         model = Job
-        fields = ('contact_email', 'title', 'description', 'city',
-                  'country', 'expiration_date')
-        exclude = ('company',)
```

Comment:
I had some errors when I didn't, but maybe they were caused by something else. I'll try again without the exlude